ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js

```diff
@@ -181,7 +181,6 @@ if (typeof window.DatePickerWidget === 'undefined') {
       }
       html += this.#clearButtonTemplate;
 
-      this.#dp = document.getElementsByClassName("datetimepicker")[0];
```

Comment:
Please add test cases and raise the PR against the dev branch

Reply:
Okay!